src/actions/accountRegistration.ts

```diff
@@ -1,26 +1,40 @@
 import axios from 'axios'
 
+export enum AccountRegistrationResult {
```

Comment:
nice thingy! but maybe you can take a look also at SWR and adopt code a bit.. I use it once alredy, documentation is small. I'm not sure with SWR it's possible to replace your code, but can be.

Reply:
After discussion, we have decided not to use SWR here